.github/workflows/publish_conda.yml

```diff
@@ -0,0 +1,18 @@
+name: publish_conda
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: publish-to-conda
+      uses: felix5572/conda-publish-action@v1.9
+      with:
+        subdir: 'conda'
+        anacondatoken: ${{ secrets.ANACONDA_TOKEN }}
```

Comment:
Have you added the token?

Reply:
Haven't now, I don't have the permission to add repo secret.  We could add the secret later?